examples/networks/networks.go

```diff
@@ -0,0 +1,120 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	client "github.com/shakenfist/client-go"
+)
+
+func printNetwork(network client.Network) {
+	fmt.Printf("UUID: %s\n", network.UUID)
+	fmt.Printf("Name: %s\n", network.Name)
+	fmt.Printf("Net Block: %s\n", network.NetBlock)
+	fmt.Printf("Provide DHCP: %t\n", network.ProvideDHCP)
+	fmt.Printf("Provide NAT: %t\n", network.ProvideNAT)
+	fmt.Printf("Owner: %s\n", network.Owner)
+	fmt.Printf("Floating Gateway: %s\n", network.FloatingGateway)
+	fmt.Printf("State: %s\n", network.State)
+	fmt.Printf("StateUpdated: %s\n", time.Unix(int64(network.StateUpdated), 0))
+	fmt.Println("")
+}
+
+func printInstance(instance client.Instance) {
```

Comment:
I can't see where this is called in this file?

Reply:
good point. cut and paste error